app/controllers/comments_controller.rb

```diff
@@ -12,9 +12,11 @@ def create
 
     if @comment.save
       @comment.update_comments_counter
-      redirect_to user_posts_path(params[:user_id]), notice: 'You have successfully created a comment'
+      redirect_to user_posts_path(params[:user_id])
+      flash[:notice] = 'You have successfully created a comment'
```

Comment:
- [ ] Flash messages do not work like expected. After creating the message page redirect but to see the message ı have to go back to the posting page. Please fix it.

Reply:
Resolved